Fix headers dependencies so that they can be included alone.

--- include/ifaddrs.h.old	2009-05-15 13:50:10 +0000
+++ include/ifaddrs.h	2009-05-15 13:50:23 +0000
@@ -28,6 +28,8 @@
 #ifndef	_IFADDRS_H_
 #define	_IFADDRS_H_
 
+#include <sys/types.h>
+
 struct ifaddrs {
 	struct ifaddrs  *ifa_next;
 	char		*ifa_name;
